docs/securing_installation.md

```diff
@@ -0,0 +1,108 @@
+
+### Tiller's Release Information
+
+For historical reasons, Tiller stores its release information in ConfigMaps. We suggest changing the default to Secrets.
+
+Secrets are the Kubernetes accepted mechanism for saving configuration data that is considered sensitive. While secrets don't themselves offer many protections, Kubernetes cluster management software often treats them differently than other objects. Thus, we suggest using secrets to store releases.
+
+Enabling this feature currently requires setting the `--storage=secret` flag in the tiller-deploy deployment. This entails directly modifying the deployment or using `helm init --override=...`, as no helm init flag is currently available to do this for you. For more information, see [Using --override](install.md#using---override).
+
+### Thinking about Charts
+
+Because of the relative longevity of Helm, the Helm chart ecosystem evolved without the immediate concern for cluster-wide control, and especially in the developer space this makes complete sense. However, charts are a kind of package that not only installs containers you may or may not have validated yourself, but it may also install into more than one namespace. 
+
+As with all shared software, in a controlled or shared environment you must validate all software you install yourself _before_ you install it. If you have secured Tiller with TLS and have installed it with permissions to only one or a subset of namespaces, some charts may fail to install -- but in these environments, that is exactly what you want. If you need to use the chart, you may have to work with the creator or modify it yourself in order to use it securely in a mulitenant cluster with proper RBAC rules applied.
+
+### gRPC Tools and Secured Tiller Configurations
+
+Many very useful tools use the gRPC interface directly, and having been built against the default installation -- which provides cluster-wide access -- may fail once security configurations have been applied. RBAC policies are controlled by you or by the cluster operator, and either can be adjusted for the tool, or the tool can be configured to work properly within the contraints of specific RBAC policies applied to Tiller. The same may need to be done if the gRPC endpoint is secured: the tools need their own secure TLS configuration in order to use a specific Tiller instance. The combination of RBAC policies and a secured gRPC endpoint configured in conjunction with gRPC tools enables you to control your cluster environment as you should.
+
+## Best Practices for Securing Helm and Tiller
+
+The following guidelines reiterate the Best Practices for securing Helm and Tiller and using them correctly. 
+
+1. Create a cluster with RBAC enabled
+2. Configure each Tiller gRPC to use TLS 
```

Comment:
I'd say "Configure each Tiller gRPC endpoint to use a separate TLS CA"

Reply:
I see what you mean here.